addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7407,7 +7407,7 @@ msgstr ""
 
 #: xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp
 msgctxt "#15049"
-msgid "The audio DSP manager has been enabled without any enabled DSP add-on. Enable at least one add-on in order to use the audio DSP functionality."
+msgid "The DSP manager has been enabled without any enabled DSP add-on. Enable at least one add-on in order to use the DSP functionality."
```

Comment:
This is for the manager itself not for the addons the addons can vary like in PVR

but OK I put it audio back.